storage_service/static/js/project.js

```diff
@@ -1,5 +1,6 @@
 $(document).ready(function() {
   var uri = $("#user-data-packages").data("uri") || "/";
+  var location = $("#user-data-packages").data("location-uuid");
```

Comment:
IIRC, it's preferably to cache the object (`$("#user-data-packages")`) because the jq constructor is expensive.

Reply:
:+1: Will fix.